evm/src/executor/inspector/cheatcodes/env.rs

```diff
@@ -191,6 +191,82 @@ fn get_recorded_logs(state: &mut Cheatcodes) -> Bytes {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct MappingSlots {
```

Comment:
I'd like to move this to a separate module, 
I know this file already lacks some structure, but let's not make it worse :)

please add a simple one line struct doc,